src/Core/Ripgrep.re

```diff
@@ -271,18 +272,35 @@ let search =
 };
 
 let findInFiles =
-    (~executablePath, ~directory, ~query, ~onUpdate, ~onComplete, ~onError) => {
-  process(
-    executablePath,
-    [
+    (
+      ~executablePath,
+      ~searchExclude,
+      ~directory,
+      ~query,
+      ~onUpdate,
+      ~onComplete,
+      ~onError,
+    ) => {
+  let excludeArgs =
+    switch (searchExclude) {
+    | [""] => []
+    | searchExclude => searchExclude |> List.concat_map(x => ["-g", "!" ++ x])
+    };
```

Comment:
We could potentially replace this with:
```
  let excludeArgs = searchExclude
    |> List.filter(str => !StringEx.isEmpty(str))
    |> List.concat_map(x => ["-g", "!" ++ x]);
```

This would have the added benefit of handling some other cases like `["", ""]`, or `["valid-path", ""]`